babel.config.js

```diff
@@ -30,7 +30,6 @@ const defaultAlias = {
   '@mui/x-tree-view': resolveAliasPath('./packages/x-tree-view/src'),
   '@mui/x-tree-view-pro': resolveAliasPath('./packages/x-tree-view-pro/src'),
   '@mui/x-internals': resolveAliasPath('./packages/x-internals/src'),
-  '@mui/material-nextjs': '@mui/monorepo/packages/mui-material-nextjs/src',
```

Comment:
We had the dependency, but it was not even used as we were aliasing it... 🤷 🤯 🙈

Reply:
some day we like to remove the alias and just rely on the dependency

Reply:
I think the point is that it is not even used

Reply:
But do you think there is a need for this dependency to exist?
I can find https://github.com/mui/material-ui/blob/1920b80bdf58e6fb1f6bb4394f55fa0a07d0faf0/docs/pages/_document.js#L3 that seems like a downstream usage, but I struggle to understand if anything is impacted by the removal of this dependency. 🤔 
A `style` element with `insertion-point-jss` id is still inserted in the DOM. 🤷

Reply:
I think it's one of the implicit dependencies coming from the docs. since the docs dependencies aren't installed automatically but we are using monorepo docs code that uses those dependencies, we need to copy them exactly from the monorepo. hopefully we can move all that usage and dependencies to the @mui/docs package some day so that it happens automatically.

Reply:
I'm merging this "optimization".
Once we further the progress on the `@mui/docs` migration I hope that the package will handle the necessary dependencies installation. 👍

Reply:
If it's a dependency of the docs workspace in core, it should probably be one of the docs workspace in X/toolpad/... as well. If it's not broken, then that's only by coincidence.
But I'm not going to die on this hill, it'll break when it wants to break 😄